web/src/About.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
```

Comment:
can you change this file to .js, so its more consistent with the rest of our things.

Reply:
done for all!